Debian pkg config file updated in snippet (#1457)
elastic/docs-content: deploy-manage/deploy/self-managed/_snippets/etc-elasticsearch.md

{{es}} loads its configuration from the `/etc/elasticsearch/elasticsearch.yml` file by default. The format of this config file is explained in [](/deploy-manage/deploy/self-managed/configure-elasticsearch.md).

> The {{distro}} package also has a system configuration file ({{pkg-conf}}), which allows you to set the following parameters:

Reviewer: I don't mind the missing code formatting too much. you could do it like that, or alter it like this: 

````suggestion
The {{distro}} package also has a system configuration file at the following path:
 
```subs=true
{{pkg-conf}}
```

In this file, you can set the following parameters:
````

no strong preference.
